app/main/routes_frontend.py

```diff
@@ -205,6 +206,22 @@ def delete_zseries_recipe():
     return 'Delete Recipe: Failed to find recipe id \"' + recipe_id + '\"', 418
 
 
+@main.route('/import_zseries_recipe', methods=['GET', 'POST'])
+def import_zseries_recipe():
+    if request.method == 'POST':
+        current_app.logger.debug('Importing Z recipes')
+        data = request.get_json()
+        guid = data['guid']

```

Comment:
This is what the guid of the device or the guid of the user profile similar to the zymatic? Would name the variable reflecting the actual model type.